django_irods/views.py

```diff
@@ -111,8 +111,35 @@ def download(request, path, rest_call=False, use_async=True, use_reverse_proxy=T
     output_path = path
 
     irods_output_path = irods_path
+
     # folder requests are automatically zipped
     if not is_bag_download and not is_zip_download:  # path points into resource: should I zip it?
+        # check for aggregations
+        if is_aggregation_request and res.resource_type == "CompositeResource":
+            try:
+                aggregation_name = path[len(res.short_id + "/data/contents/"):]
```

Comment:
Is it guaranteed that ```/data/contents/``` would be part of the path? Perhaps a check is needed here?

Reply:
It definitely should be, but I'll add a check.